app-backend/common/src/main/scala/Authentication.scala

```diff
@@ -1,12 +1,14 @@
 package com.azavea.rf.common
 
+import scala.util.{Success,Failure}
```

Comment:
General rule for imports (the one that GeoTrellis uses):

```scala

// Imports from the project, alphabetical
import com.azavea.rf._
import com.azavea.rf.foo._

// Imports from 3rd party libs, alphabetical
import com.what._
import org.huh._
import typlevel.funky.{Stuff1, Stuff2}

// Imports from scala and java, alphabetical
import java.io._
import scala.util.{Success, Failure}
```

Not sure if this project is following that exact structure, but if there's another ordering I'm not sure these fit the pattern.